ajax/libs/isomorphic-fetch/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "isomorphic-fetch",
+  "description": "Isomorphic WHATWG Fetch API, for Node & Browserify",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/matthew-andrews/isomorphic-fetch.git"
+  },
+  "author": "Matt Andrews <matt@mattandre.ws> (http://mattandre.ws/)",
+  "license": "MIT",
+  "filename": "fetch-npm-node.js",
```

Comment:
@pvnr0082t But I don't need to create a `min` version manually right? Since this is adding via single package.json.